### DISCUSSION TITLE
Reference on Using Git

### DISCUSSION THREAD

**Discussion by HannahStrimpel** [frc5567/2023-Robot#214] [Team Posts]
Here is a link to how to use Git to clone repositories and create/use your own branch in VS Code.

https://onedrive.live.com/view.aspx?resid=8237DE166D19E9B4!751261&ithint=file%2cdocx&app=Word&authkey=!AAh40RuGN_IY6dA

**Comment by coverbeek** [comment]
Thank you for posting, @HannahStrimpel! We may want to put this in the Documents and Resources repository for future reference!
